src/instructlab/model/chat.py

```diff
@@ -30,7 +30,7 @@
 # Local
 from ..utils import get_sysprompt, http_client
 
-logger = logging.getLogger(__name__)
+ilab_logger = logging.getLogger(__name__)
```

Comment:
Why the rename?

Reply:
This is because linter report warning/error on line 530, where start_prompt use the same name for its argument, which should be avoided. (I might add code to suppress pylint, which I should remove). Changing the global variable is safer and quicker fix.